Chris dokumentation (#58)
jbathmann/pyMANGA: web/content/en/docs/getting_started/Installation/Preparation/Windows.md

<figure>
<img src="/de/static/oeffnen_der_Eingabeaufforderung.jpg">
<figcaption><font size = "1"><i><b>Figure 1:</b>open the command prompt.</i></font></figcaption>
</figure><p>

Now the following modules *numpy*, *vtk*, *lxml* and *matplotlib* must be installed. We start with the module *numpy*. Type the code shown in the prompt to install the module (see Figure 2).

	• py -3.7 -m pip install numpy						     [1]

<figure>
<img src="/de/static/Beispielhafte_Installation_des_Moduls_numpy.jpg">
<figcaption><font size = "1"><i><b>Figure 2:</b> Exemplary installation of the numpy module.</i></font></figcaption>
</figure><p>

Do the same for the other three modules with the following code

	• py -3.7 -m pip install vtk						     [2]
	• py -3.7 -m pip install lxml					  	     [3]
	• py -3.7 -m pip install matplotlib					     [4]

Note: If the prompt says that *pip* is not up to date, you can use upgrade *pip* to update it. However, this is not mandatory.

> To explain what you have entered: *py* means you are calling Python. Where -3.7 is the version you are using. *-m* means you are calling a module, in this case *pip*, which is used to install other modules. Finally, the module name of the module to be installed follows. Now the preparations for using the compiler are finished. As a next step you have to download the program MANGA, if you haven't done so already. To do this, go to the following website [Link](https://github.com/jbathmann/pyMANGA/ "https://github.com/jbathmann/pyMANGA/") and download the program as a zip file and save it on your computer (see Figure 3).

Reviewer: ```suggestion
Some explenations: *py* means you are calling Python. Where -3.7 is the version you are using. *-m* means you are calling a module, in this case *pip*, which is used to install other modules. Finally, the module name of the module to be installed follows. Now the preparations for using the compiler are finished. As a next step you have to download the program MANGA, if you haven't done so already. To do this, go to the following website [Link](https://github.com/jbathmann/pyMANGA/ "https://github.com/jbathmann/pyMANGA/") and download the program as a zip file and save it on your computer (see Figure 3).
```
